docker/debug/Dockerfile

```diff
@@ -5,7 +5,7 @@ ARG TARGETARCH
 ENV GOPATH /go
 RUN apk add --update --no-cache ca-certificates make git
 #Once go-delve adds support for s390x (see PR #2948), remove this entire conditional.
-RUN if [[ "$TARGETARCH" != "s390x" ]] ; then \
+RUN if [[ "$TARGETARCH" != "s390x" ] || [ "$TARGETARCH" != "ppc64le" ]] ; then \
```

Comment:
Is there a similar PR for delve to address ppc64le? If so, we need to list it here, as we are doing with s390x

Reply:
Yes, currently the same is discussed via https://github.com/go-delve/delve/issues/1564 
So do you want this to go inside the PR, just like the way it is done for s390x?

Reply:
yes

Reply:
Ok, added the required details into the PR.